Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -58,6 +65,48 @@ YUI.add('ez-locationviewlocationstabview', function (Y) {
                 content: this.get('content')
             });
         },
+
+        /**
+         * Tap event handler on the `Add location` button. It fires the
+         * `createLocation` event
+         *
+         * @method _addLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _addLocation: function (e) {
+            /**
+             * Fired when the user clicks on `Add location` button
+             *
+             * @event createLocation
+             * @param {eZ.Content} content the content for which locations will be created
+             * @param {Function} afterCreateCallback callback function that will be called after
+             *                   creating location(s)
+             */
+            this.fire('createLocation', {
+                content: this.get('content'),
+                afterCreateCallback: Y.bind(this._afterCreateLocationCallback, this)
+            });
+        },
+
+        /**
+         * After create location callback function. It fires `loadLocations` event
+         * for refresh the view.
+         *
+         * @method _afterCreateLocationCallback
+         * @protected
+         */
+        _afterCreateLocationCallback: function () {
```

Comment:
that's a bad name, since you can call it to refresh the view, why don't you call this method `_refresh` ?